selene-lib/src/rules/duplicate_keys.rs

```diff
@@ -150,6 +150,8 @@ impl Visitor<'_> for DuplicateKeysVisitor {
                     };
                     self.check_field(&mut declared_fields, key, field_range)
                 }
+
+                other => panic!("unknown node {:?}", other),
```

Comment:
We shouldn't panic! The point of the `non_exhaustive` is so we can add new nodes without having to update consumers.